tests/cephfs/cephfs_bugs/test_mds_inject_health_dummy.py

```diff
@@ -0,0 +1,154 @@
+        output = json.loads(out)
+        deamon_dict = fs_util.filter_daemons(output, "mds", fs_name)
+        for node in deamon_dict:
+            for hostname, deamon in node.items():
+                if hostname in standby_reply_mdss_obj[0].hostname:
+                    standby_reply_mdss_obj[0].exec_command(
+                        sudo=True,
+                        cmd=f"cephadm shell -- ceph daemon {deamon} config set mds_inject_health_dummy true",
+                    )
+        fuse_mount_dir = "/mnt/fuse_" + "".join(
+            secrets.choice(string.ascii_uppercase + string.digits) for i in range(5)
+        )
+        fs_util.fuse_mount(
+            [client1],
+            fuse_mount_dir,
+            new_client_hostname="admin",
+            extra_params=f" --client_fs {fs_name}",
+        )
+        client1.exec_command(
+            sudo=True,
+            cmd=f"cd {fuse_mount_dir};"
+            f"mkdir subdir;"
+            f"dd if=/dev/urandom of=subdir/sixmegs bs=1M conv=fdatasync count=6 seek=0;",
+        )
+        time.sleep(60)
+
+        sstanby_reply_after, rc = client1.exec_command(
+            sudo=True, cmd="ceph fs dump -f json"
+        )
+
+        info_after = get_info(json.loads(sstanby_reply_after), fs_name)
+        standby_replay_dict_after = {
+            key: value
+            for key, value in info_after.items()
+            if "standby-replay" in value["state"]
+        }
+        log.info(standby_replay_dict_before)
+        log.info(standby_replay_dict_after)
+        gid_before = [value["gid"] for value in standby_replay_dict_before.values()]
+        gid_after = [value["gid"] for value in standby_replay_dict_after.values()]
+
+        if gid_before != gid_after:
+            log.error(
+                f"gid values are not matching before : {gid_before} and after : {gid_after}"
+            )
+            return 1
+
+        return 0
+
+    except CommandFailed as e:
+        log.error(e)
+        log.error(traceback.format_exc())
+        return 1
+    except Exception as e:
+        log.error(e)
+        log.error(traceback.format_exc())
+        return 1
+    finally:
+        client1.exec_command(sudo=True, cmd=f"umount {fuse_mount_dir}", check_ec=False)
+        client1.exec_command(sudo=True, cmd=f"rm -rf {fuse_mount_dir}", check_ec=False)
+        client1.exec_command(
+            sudo=True, cmd=f"ceph fs set {fs_name} allow_standby_replay 0"
+        )
+        client1.exec_command(
+            sudo=True, cmd="ceph config set mon mon_allow_pool_delete true"
+        )
+        fs_util.remove_fs(client1, fs_name)
+
+
+@retry(CommandFailed, tries=3, delay=30)
+def wait_for_standby_reply_mds(fs_util, client1, fs_name):
+    standby_reply_mds = fs_util.get_standby_replay_mdss(client1, fs_name=fs_name)
+    if not standby_reply_mds:
+        raise CommandFailed("No Stanby Reply MDS")
+
+
+def get_info(json_output, fs_name):
```

Comment:
rename to "get_filesystem_info" ?

Can this be moved to utils file ?

Reply:
this is intended for this TC. so I kept here only